woocommerce/woocommerce-ios
# [Woo POS] Use separate views for different payment states

Closes: #13451


## Description


Replace `POSCardPresentPaymentMessageView` with separate views for each payment state. It will allow for more granular control and configuration of each of the views. 

I also introduced `PointOfSaleCardPresentPaymentLayout` `enum` for sharing certain layout properties between views for consistency.

For now, for each of the views I copied `POSCardPresentPaymentMessageView` and removed subviews that the views weren't currently using. Since we haven't yet implemented final layout for most of these views, the exact implementation is not important.

## Testing information

1. Open Woo -> Menu -> Point of Sale Mode
2. Keep reader disconnected
3. Proceed to check out
4. Confirm 'reader disconnected' view appears
6. Connect the reader
7. Confirm correct views are shown throughout payment process

## Screenshots


https://github.com/user-attachments/assets/b016aa22-e566-454a-9b24-cce2b8c98aca

---
